src/main/assembly/dist/cfg/application.properties

```diff
@@ -38,17 +38,27 @@ users.ldap-url=ldap://localhost
 users.ldap-parent-entry=ou=users,ou=easy,dc=dans,dc=knaw,dc=nl
 users.ldap-user-id-attr-name=uid
 
-# Settings required to fetch user data when we no longer have its password.
+#
+# Settings required to fetch user data when we no longer have the user's password.
+#
 users.ldap-admin-principal=cn=ldapadmin,dc=dans,dc=knaw,dc=nl
 users.ldap-admin-password=changeme
 
+#
+# Base URL of the easy-pid-generator service to use for minting DOIs.
+#
 pids.generator-service=http://localhost:20140/
 
-
-# Lifetime for cookie with name scentry.auth.default.user that contains the jwt after providing valid credentials.
-# In seconds, 60*60=3600 is one hour
+#
+# Lifetime for JSON Web Token cookie (named "scentry.auth.default.user") in seconds.
+# (Example: 3600 ( = 60 * 60 = is one hour ).
+#
 auth.cookie.expiresIn=3600
 
-# Pick a value starting with H, see fromString in https://github.com/pauldijou/jwt-scala/blob/master/core/common/src/main/scala/JwtAlgorithm.scala
+#
+# Algorithm and secret key to use for JSON Web Token. Algorithm can be any of the values supported by jwt-scala, see fromString in:
+#
+# https://github.com/pauldijou/jwt-scala/blob/master/core/common/src/main/scala/JwtAlgorithm.scala
+#
```

Comment:
@jo-pol : in the previous version of this comment it said a value starting with an H must be chosen? Why are the other values not valid?

Reply:
Only hmac is imlpemented:
https://github.com/DANS-KNAW/easy-deposit-api/blob/5340ca35037c255aa76bdc7bff89113bb86ca711/src/main/scala/nl.knaw.dans.easy.deposit/authentication/TokenSupport.scala#L33-L44

Reply:
The others a asymmetric algorithms and require two keys to encrypt/decrypt. We need only one key as cookies are both encrypted and decrypted on the server.

Reply:
OK, thanks. I'll update the documentation comment. This way the future configurator (possibly you ;) will have a bit more understanding of the why of things.